klojure-catalog/src/main/clojure/klojure/catalog/core.clj

```diff
@@ -68,11 +70,21 @@
       (.setAttribute impl (name k) v))
     impl))
 
+(defn- bundle-cxt
+  "Returns the bundle context for the nREPL bundle."
+  []
+  (.. FrameworkUtil (getBundle RT) getBundleContext))
+
+(defn- get-services [name]
```

Comment:
Is there no way to use the `osgi.core` namespace?

Reply:
Not unless I split that out because it's shared between two modules now but it didn't seem worth it. And on top of that I'd need to copy the resources to both repls, I think.